src/core/deltafilewrapper.cpp

```diff
@@ -641,17 +613,20 @@ void DeltaFileWrapper::addPatch( const QString &localLayerId, const QString &sou
   if ( tmpOldAttrs.length() > 0 || tmpNewAttrs.length() > 0 )
```

Comment:
could `tmpOldAttrs` be renamed to `changedOldAttrs`? IIUC that's what it refers to

Reply:
@m-kuhn , I've tried to avoid variable renaming noise so the logic tweak is easier to review.

Reply:
Good choice.
If it makes sense for you in principle, this can also be done in a follow up PR.

Reply:
@m-kuhn , yep, that's fair.